modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -446,6 +448,13 @@
   [_ entity-name]
   (str/replace entity-name "/" "//"))
 
+(defmethod sql-jdbc.describe-table/get-table-pks :oracle
+  [_driver ^Connection conn _ table]
+  (let [^DatabaseMetaData metadata (.getMetaData conn)]
+    (into #{} (sql-jdbc.sync.common/reducible-results
+               #(.getPrimaryKeys metadata nil nil (:name table))
```

Comment:
How strong is our conviction that we don't want to be passing `(:schema table)` here?

Reply:
I know Oracle supports schemas